contracts/externals/strings.sol

```diff
@@ -50,7 +50,9 @@
  *      corresponding to the left and right parts of the string.
  */
 
-pragma solidity ^0.5.0;
+// SPDX-License-Identifier: ApacheV2
+
+pragma solidity >=0.5.0;
```

Comment:
It seems that the strings.sol library has been updated to 0.6.0 by OpenZeppelin https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Strings.sol